contrib/awslambda/python/tests/python/pants_test/contrib/awslambda/python/test_python_awslambda_integration.py

```diff
@@ -22,7 +22,11 @@ def test_awslambda_bundle(self):
         'GLOBAL': {
           'pants_distdir': distdir,
           'pythonpath': ['%(buildroot)s/contrib/awslambda/python/src/python'],
-          'backend_packages': ['pants.backend.python', 'pants.contrib.awslambda.python'],
+          'backend_packages': [
```

Comment:
It looks like there is precedent for "default installing" `src/python/pants/core_tasks/register.py`, so that they are always available, regardless of the backend setting. I think that that would be reasonable to do here as well?

Reply:
I'm not sure what you're referring to exactly. The force-push nuked the context.

Reply:
Yea, I ended up convincing myself that it was the right approach. Now I just need to deal with the fallout.